.travis.yml

```diff
@@ -11,3 +11,18 @@ script: script/cibuild
 branches:
   only:
     - master
+jobs:
+  include:
+    -
+      if: branch = master AND (type != pull_request OR type != api)
```

Comment:
```ruby
(type != pull_request OR type != api)
```

wat?
Isn't this condition always true?

Reply:
I was going for commits pushed to `master` (merge commits from PRs, though I now realize that JekyllBot adds `[ci skip]` to the merge commits)

Reply:
What I was getting at is that you mean `AND` rather than `OR`

Reply:
the `type:` [can be](https://docs.travis-ci.com/user/conditions-v1#Integration) either of `push`, `pull_request`, `api`, `cron`. I felt using `OR` to be more optimal. But if you feel `AND` would be better, I'm fine with that..

Reply:
```ruby
(type != pull_request OR type != api)
```

The `OR` statement will only evaluate to false if both `type != pull_request` and `type != api` are false (because that is what disjunctions are for). How is this possible? What value of `type` will possibly make both of these statements false?

You may as well replace the whole condition with
```ruby
 branch = master AND (true)
```

Reply:
Think about it this way.

We can use DeMorgan's Law to write the following, logically equivalent statement:

```ruby
!(type == pull_request AND type == api)
```

It is _not possible_ for `type == pull_request` and `type == api` to both be true at the same time, so the `AND` will always be false, and `!(false)` is `true`.

Reply:
@pathawks I concede. I've fixed the error and replaced the operators with "aliases" to resemble the operators used in Ruby